tests/ci/run_accp_basic_tests.sh

```diff
@@ -3,11 +3,26 @@ set -exo pipefail
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-echo "Testing ACCP standard release with corretto 11."
-./gradlew release
+# Parse and check which JDK version we're testing upon.
+version=$($TEST_JAVA_HOME/bin/java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1)
 
-echo "Testing ACCP standard release with corretto 8."
-./gradlew -DTEST_JAVA_HOME=/usr/lib/jvm/java-1.8.0-amazon-corretto test
+# The JDK version should be least 10 for a regular ACCP build. We can
+# still test on older versions with the TEST_JAVA_HOME property.
+if (( "$version" <= "10" )); then
+	./gradlew -DTEST_JAVA_HOME=$TEST_JAVA_HOME test
+	exit 0
```

Comment:
i don't see `-e` set at the top of this script... shouldn't you `exit $?` to pick up the result of your gradle invocation?

Reply:
`-e` is set above the copyright comments, but either way, using `exit $?` is much cleaner.

Reply:
ah whoops, i forgot to expand the diff boundary. thanks.